universal_transfer_operator/src/universal_transfer_operator/data_providers/filesystem/google/cloud/gcs.py

```diff
@@ -59,9 +59,10 @@ def transport_params(self) -> dict:
     def paths(self) -> list[str]:
         """Resolve GS file paths with prefix"""
         url = urlparse(self.dataset.path)
+        prefix = url.path[1:]
```

Comment:
Are there any scenarios where this would fail? Not sure if we need a test around this.

Reply:
@dimberman it shouldn't fail. 
<img width="139" alt="Screenshot 2023-02-27 at 4 16 49 PM" src="https://user-images.githubusercontent.com/13021213/221543769-9da88039-169f-4a4f-a005-049de31e3983.png">